six/modules/c++/six.sidd/include/six/sidd/SFA.h

```diff
@@ -68,16 +70,18 @@ struct SFAPoint : public SFAGeometry
     double x, y, z, m;
 
     SFAPoint() :
-        SFAGeometry(TYPE_NAME)
+        SFAGeometry(TYPE_NAME), x(0), y(0),
+        z(::six::Init::undefined<double>()),
+        m(::six::Init::undefined<double>())
     {
-        x = 0;
-        y = 0;
         z = ::six::Init::undefined<double>();
```

Comment:
Do you mean one line for both x and y, or x on one line and y on another?

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Friday, February 12, 2016 3:20 PM
To: ngageoint/six-library <six-library@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>
Subject: Re: [six-library] Fix object slicing issue with Annotations, clean up some warnings (#48)


In six/modules/c++/six.sidd/include/six/sidd/SFA.h<http://cp.mcafee.com/d/k-Kr6xEq6zqb3XXdTCbL6XCSm6hMUQsLLCTNPb3a9EVd79J6XRTD3hPsQsL6QkmkkTPhOUM-r4qMSpfosH-ME2BkdrFOVJl3mWsKru7uPV1N_HYO-qerFFLtuVtdB5fxP3BNPXTkhjmKCHtBDBgY-F6lK1FJ4SCrLOb2rPUV5xcQsCXCOsVHkiP2cFASO7bV0zF7WOxfai3OVYy4tymNpIY_67wcgBiPtaiH0ly-Ced7bbb3VK5o-Ium1h6kOrp3BPr9EV7f6MnWhEwdbt91Vs-h2eNb6BQQg2YwYQgl82fSCq8a7OJa14QgiFhLPh0oBcCq8124aPSsGME0pEwIv-Md43qNEwFlnPh0oSc9g8Qd41EE0NxUQkCTzhOr-yaQXR5bS5H0>:

>      {

> -        x = 0;

> -        y = 0;

>          z = ::six::Init::undefined<double>();

z and m can be removed from the constructor body. Also, put x and y on their own lines

—
Reply to this email directly or view it on GitHub<http://cp.mcafee.com/d/2DRPoOd2gQrhovvpKYNtUTsSOMOe76zBZYS-epophd79EVdETuKYUqerCzBUSyyOyC-qen67Pozm6P9X3BvS50kGxHtendGEqTjBPrMXSv8efZvCnPhPtddXHTbFIEFYeosKevuWyaqRQRrIIYG7DR8OJMddFCQPt-hojuv78I9CzATsSjDdqymohBcCSgVv84t8_mk9VigunfAgzIiSbdDDUMZzIz_E1AnQNNEVppovdMH7RzOMa8OCjr8sKrpd78VUS2_id41FrF8fbDO8hS9oQKCy0nA7Cy2F0h-QPh1g-lFg8Cy2lad-q834FAPh08gxmuPBm503d45z_S1Ewrmd45aG-q836Nxa16xEwd506cf6yASYqejuA35bMREXaCT>.


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.